docs/.vuepress/config.js

```diff
@@ -38,6 +38,10 @@ module.exports = {
       index: "cosmos-sdk"
     },
     versions: [
+      {
+        "label": "v0.40",
+        "key": "v0.40"
+      },
```

Comment:
just a note: I'd say 0.40 docs are 90% done, however, a lot of them are not on the `release/v0.40.x` branch yet, and need to be backported (will happen most likely in this week and next week).

If you think it'd be better to wait until all docs are updated on `release/v0.40.x` first, we can also wait a bit for this PR.